app/src/lib/dispatcher/dispatcher.ts

```diff
@@ -1173,4 +1173,25 @@ export class Dispatcher {
   public executeCompare(repository: Repository, action: CompareAction) {
     return this.appStore._executeCompare(repository, action)
   }
+
+  /**
+   * Increments the `mergeIntoCurrentBranchMenuCount` metric
+   */
+  public recordMenuInitatedMerge() {
```

Comment:
minor ⛏ for a few of these, s/`Initated`/`Initiated`
(we should probably do a find & replace because it's in this file a few times)